pandas/tests/io/parser/conftest.py

```diff
@@ -1,14 +1,15 @@
 import os
+from typing import List, Union
 
 import pytest
 
 from pandas import read_csv, read_table
 
 
 class BaseParser:
-    engine = None
+    engine = None  # type: Union[str, None]
```

Comment:
Use `Optional[str]` instead of `Union[str, None]`

Reply:
Updated with bbb1fc6